third_party/terraform/website/docs/version_2_upgrade.html.markdown

```diff
@@ -60,11 +87,16 @@ Upgrade topics:
 
 ## Provider Version Configuration
 
-!> **WARNING:** This topic is placeholder documentation until version 2.0.0 is released later this year.
+-> Before upgrading to version 2.0.0, it is recommended to upgrade to the most
+recent version of the provider (1.20.0) and ensure that your environment
+successfully runs [`terraform plan`](https://www.terraform.io/docs/commands/plan.html)
+without unexpected changes or deprecation notices.
 
--> Before upgrading to version 2.0.0, it is recommended to upgrade to the most recent version of the provider (1.19.1) and ensure that your environment successfully runs [`terraform plan`](https://www.terraform.io/docs/commands/plan.html) without unexpected changes or deprecation notices.
-
-It is recommended to use [version constraints when configuring Terraform providers](https://www.terraform.io/docs/configuration/providers.html#provider-versions). If you are following that recommendation, update the version constraints in your Terraform configuration and run [`terraform init`](https://www.terraform.io/docs/commands/init.html) to download the new version.
+It is recommended to use [version constraints](https://www.terraform.io/docs/configuration/providers.html#provider-versions)
+when configuring Terraform providers. If you are following that recommendation,
```

Comment:
nit: we should probably tell people what to do if they _aren't_ following that recommendation (namely, `terraform init -upgrade`).

Reply:
👍